src/app/wizard/step/provider-settings/provider/basic/openstack/component.ts

```diff
@@ -101,12 +104,16 @@ export class OpenstackProviderBasicComponent extends BaseFormValidator implement
     private readonly _presets: PresetsService,
     private readonly _clusterService: ClusterService,
     private readonly _datacenterService: DatacenterService,
+    private readonly _appConfigService: AppConfigService,
```

Comment:
Please add the import for `AppConfigService`.
`import {AppConfigService} from '@app/config.service';`

Reply:
:-O That comes from cherrypicking without testing again. I'll fix asap, thanks.